### PR TITLE
07 api from new version of script

### PR DESCRIPTION
**Pull request by RichieEscarez**
Updated version of 07 API using revised API build script (#1559)

### CODE REVIEW

File: docs/reference/eventing/eventing-contrib-resources.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ Kubernetes core/v1.SecretKeySelector
 <p>GcpCredsSecret is the credential to use to poll the GCP PubSub Subscription. It is not used
 to create or delete the Subscription, only to poll it. The value of the secret entry must be
 a service account key in the JSON format (see
-<a href="https://cloud.google.com/iam/docs/creating-managing-service-account-keys)">https://cloud.google.com/iam/docs/creating-managing-service-account-keys)</a>.</p>
+<a href="https://cloud.google.com/iam/docs/creating-managing-service-account-keys">https://cloud.google.com/iam/docs/creating-managing-service-account-keys</a>.</p>
```
Comment (ahmetb) [inline docs/reference/eventing/eventing-contrib-resources.md]: I recommend making a pr to the .go file for this so you don’t have to remember next time.

**Comment by ahmetb** [comment]
I don’t know if Netlify stages these somewhere, but the diff looks good I think.

File: docs/reference/eventing/eventing-contrib-resources.md (hunk 1 above)
Comment (RichieEscarez) [reply docs/reference/eventing/eventing-contrib-resources.md]: Thanks! Done: https://github.com/knative/eventing-contrib/pull/480

**Comment by RichieEscarez** [comment]
> I don’t know if Netlify stages these somewhere, but the diff looks good I think.

No not yet. Netlify has that as a built feature if we use only a single repo (ie our knative/website repo auto builds previews) but since our docs repo is separate from our Netlify site code, I need to add some type of preview build page (which im slowly working on).

**Comment by ahmetb** [comment]
Yaay!

**Pull request merged**